lib/trino-filesystem-s3/src/test/java/io/trino/filesystem/s3/TestS3FileSystemAwsS3.java

```diff
@@ -68,4 +78,32 @@ private static String environmentVariable(String name)
     {
         return requireNonNull(System.getenv(name), "Environment variable not set: " + name);
     }
+
```

Comment:
@pettyjamesm we'll need to run the PR with secrets to ensure that it passes the cloud CI tests

```
/test-with-secrets sha=....
```

Reply:
Added the magic comment